docs/deployment/resource-providers/standalone/docker.md

```diff
@@ -288,13 +335,11 @@ There are several ways in which you can further customize the Flink image:
 * add other libraries to `/opt/flink/lib` (e.g. Hadoop)
 * add other plugins to `/opt/flink/plugins`
 
-See also: [How to provide dependencies in the classpath]({% link index.md %}#how-to-provide-dependencies-in-the-classpath).
-
 You can customize the Flink image in several ways:
 
 * **override the container entry point** with a custom script where you can run any bootstrap actions.
-At the end you can call the standard `/docker-entrypoint.sh` script of the Flink image with the same arguments
-as described in [how to run the Flink image](#how-to-run-flink-image).
+  At the end you can call the standard `/docker-entrypoint.sh` script of the Flink image with the same arguments
+  as described in [supported deployment modes](#deployment-modes).
 
   The following example creates a custom entry point script which enables more libraries and plugins.
   The custom script, custom library and plugin are provided from a mounted volume.
```

Comment:
The linewraps look weird with the comments being on the same line as the commands in the code below. Can't we move the comments in front of the command on a separate line to avoid having linewraps?